site_scons/state.py

```diff
@@ -74,6 +75,14 @@ def _getBinPath(binName, msg=None):
         log.debug("Found %s here %s" % (binName, binFullPath))
         return binFullPath
 
+def _getEupsVersion():
+    binName = 'pkgautoversion'
+    try:
+        version = subprocess.check_output(['pkgautoversion'])
+    except:
```

Comment:
It's better to check for specific exception types if you can. Currently this will catch everything, even syntax errors in the above line.